patched-vscode/extensions/code-editor-idle-extension/README.md

```diff
@@ -0,0 +1,3 @@
+# Code Editor Idle Extension
```

Comment:
can we also add a CONTRIBUTING.md to document how to make changes to this extension and importantly how to test after making changes